http-client/src/test/java/io/airlift/http/client/jetty/TestJettyHttpsClient.java

```diff
@@ -75,7 +76,7 @@ public void tearDownHttpClient()
         }
     }
 
-    @Test(expectedExceptions = {SSLHandshakeException.class, EOFException.class})
+    @Test(expectedExceptions = {SSLHandshakeException.class, EOFException.class, ClosedChannelException.class})
```

Comment:
Well, the exception they're throwing now is entirely inappropriate and misleading. It's unlikely anyone would be able to diagnose the condition from that. But that's a bug in Jetty.